reddwarf/types/agora.py

```diff
@@ -0,0 +1,44 @@
+from typing import TypedDict, List, Optional
+from enum import IntEnum
+
+IncrementingId = int
```

Comment:
I don't think it should necessarily be incrementing.
It's up to the library consumer to uniquely name the statements and the participants!
At Agora we create unique string for this, so that the URL looks nice:
https://agoracitizen.network/feed/conversation/JYPGdw?opinionSlugId=GC3C-A

conversation_id is "JYPGdw" and statement_id "GC3C-A"

We obviously also have internal database ID but this is highly coupled with our intternal infrastructure whereas this "slug_ids" are meant to be permanent identifiers. And they are string!

Reply:
This will take a little fiddling, but totally makes sense :)